### PR TITLE
release: 0.13.0

### PR DESCRIPTION
**Pull request by eitsupi**

### CODE REVIEW

**Comment by eitsupi** [comment]
@etiennebacher I want to merge this first. Could you take a look at this?

File: NEWS.md [hunk 1]
```diff
@@ -19,13 +21,23 @@
...
+    Use the `$collect()` method after `$execute()` or `as_polars_df` to get the
     result as a `DataFrame`. (#719)
--   The argument `name_generator` of `$list$to_struct()` is renamed `fields` 
+-   The argument `name_generator` of `$list$to_struct()` is renamed `fields`
     (#724).
 -   The S3 method `[` for the `$list` subnamespace is removed (#724).
+-   The option `polars.df_print` has been renamed `polars.df_knitr_print` (#726).
 
-### What's changed
+### Deprications
```
Comment (etiennebacher) [inline NEWS.md:31]:
```suggestion
### Deprecations
```

**Review by etiennebacher — APPROVED** [review]
Sorry I didn't your message and merged two small PR before. This one looks good, just one typo

**Comment by eitsupi** [comment]
The order of the release commit and post-release fix commits have been swapped. Well, it is not a big problem.

**Comment by eitsupi** [comment]
@etiennebacher I would like to remove the setting that forces approval as it is ridiculous for this to happen.
I think there is a significant disadvantage to forcing approval because the number of maintainers is currently small.

**Comment by etiennebacher** [comment]
We can remove this so that we don't have to wait for tiny PR, but it would be good to still wait for approval for medium to large PR

**Comment by eitsupi** [comment]
Ok. This time it was my overconfidence in GitHub notifications. I should have drafted all other PRs so that they could not be merged.
The settings will be left as they are for now.

**Pull request merged**